packages/admin-e2e-tests/src/pages/AnalyticsOverview.ts

```diff
@@ -93,11 +96,12 @@ export class AnalyticsOverview extends Analytics {
 	async addSection( sectionTitle: string ) {
 		await this.page.waitForSelector( "button[title='Add more sections']" );
 		await this.page.click( "button[title='Add more sections']" );
-		// Delaying this, as there is a slight transition for the Add section button to show up.
-		await waitForTimeout( 100 );
 		await this.page.waitForSelector(
 			`button[title='Add ${ sectionTitle } section']`
 		);
+		await waitUntilElementStopsMoving(
+			`button[title='Add ${ sectionTitle } section']`
```

Comment:
Having this `button[title='Add ${ sectionTitle } section']` string copied so many times in this function makes it harder to read. Can you pull this out into a const that is used everywhere needed, instead?